streams/README.md

```diff
@@ -82,22 +85,30 @@ Arrays.stream(persons)
     ```bash
     ./streams 10000 200
     ```
+    ```
+    Iteration 20 finished in 62 milliseconds with checksum e1d39d010d08cf01
+    TOTAL time: 1279
+    ```
     Profiles collected from this run are now stored in the _default.iprof_ file.
     
     Note that we run the profiling with a much smaller data size.
     
 4. Now you can use the collected profiles to build an optimized native image:
     ```bash
-    $JAVA_HOME/bin/native-image --pgo Streams
+    $JAVA_HOME/bin/JAVA_HOME/bin/native-image --pgo Streams
```

Comment:
Why  `JAVA_HOME/bin/` is duplicated?

Reply:
I probably inserted it by accident, I'll fix it.